controller/gui/style.css

```diff
@@ -85,3 +85,20 @@ dl > dt {
 
     height: 90vh;
 }
+
+/* Network */
+
+.d-Network__Refresh {
```

Comment:
For future UI consistency, probably best to give this a generic name so that the same button style could be used in other, similar places in the controller.

Reply:
> For future UI consistency, probably best to give this a generic name so that the same button style could be used in other, similar places in the controller.

Right, I’ll make a pass in another PR, there are already some buttons with the default style.